.github/workflows/env_vars.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         envkey_DATABASE_URL_SQLALCHEMY: ${{ secrets.DATABASE_URL_SQLALCHEMY }}
         envkey_DATABASE_URL_SQLALCHEMY_TEST: ${{ secrets.DATABASE_URL_SQLALCHEMY_TEST }}
         envkey_LOCALHOST_DATABASE_URL_SQLALCHEMY: ${{ secrets.LOCALHOST_DATABASE_URL_SQLALCHEMY }}
-        envkey_ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+        envkey_ENVIRONMENT: local
```

Comment:
Since I set "environment" in the settings to default to "local", another approach is to remove it altogether from .env generation if we really don't want this to be in plain text. However, since the pattern matching is the code is in plain text I don't see much of a downside to this.